src/gui/shell.cpp

```diff
@@ -117,6 +117,11 @@ bool Shell::setGuiFont(const QString& fdesc, bool force)
 	return ok;
 }
 
+void Shell::neovimOpenFile(const QString &file)
+{
+  m_nvim->neovimObject()->vim_command(QByteArray::fromStdString("e " + file.toStdString()));
+}
```

Comment:
Add a check for neovim connection before using neovimObject

'''
if (!m_nvim || !m_nvim->neovimObject()) {
		return;
}
'''